doc/source/train/examples/lightning/lightning_cola_advanced.ipynb

```diff
@@ -217,75 +216,82 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Configure your LightningTrainer\n",
+    "## Define your Training Function\n",
     "\n",
-    "Define a LightningTrainer with necessary configurations, including hyper-parameters, checkpointing and compute resources settings. \n",
+    "Define a function that includes all your lightning training logics. This function will be launched by {class}`TorchTrainer <ray.train.torch.TorchTrainer>` on each worker in parallel. As discussed in {ref}`Lightning Quick Start Guide <train-pytorch-lightning>`, to enable distributed training with Ray Train, we need to inject several utilities(`RayDDPStrategy`, `RayLightningEnvironment`, `RayTrainReportCallback`) to the lightning's trainer.\n",
```

Comment:
Can you put the details, in particular about the utilities & the Ray Data stuff, after the code block?

That way the sections as:
- Overview of code for training function
- Code example
- Details